console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/DateUtils.java

```diff
@@ -13,15 +13,14 @@
 
 import java.util.Date;
 
-import com.google.gwt.i18n.client.TimeZone;
-
 import com.google.gwt.i18n.client.DateTimeFormat;
 
 public class DateUtils {
 
 //    private static final ConsoleMessages MSGS = GWT.create(ConsoleMessages.class);
 
     private static final String NONE = "None";
```

Comment:
Can you please localize this by using the Console Messages?

Reply:
@ct-goranpalibrk,

@Coduz meant to localize line 22 (`NONE`), not 23 (`FORMAT`). I fixed this in 551cde5.